components/Comments/Comment.vue

```diff
@@ -187,6 +184,7 @@
       startEdit () {
         this.fetchById(this.comment._id)
           .then((res) => {
+            console.log(res.content)
```

Comment:
No console output

Reply:
sorry

Reply:
Nothing to say sorry for! When I say `No this or that` I don't want to be rude. I just point out everything that I can spot on a first glimpse.